BlueprintUICommonControls/Tests/Sources/UserInteractionEnabledTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UserInteractionEnabledTests.swift
+//  BlueprintUICommonControls-Unit-Tests
+//
+//  Created by Noah Blake on 3/15/20.
+//
+
+import XCTest
+import BlueprintUI
+import BlueprintUICommonControls
+
+final class UserInteractionEnabledTests: XCTestCase {
+    func test() throws {
+        let userInteractionEnabledView: (Bool) throws -> UIView = { isUserInteractionEnabled in
+            let wrapped = UserInteractionEnabled(isUserInteractionEnabled, wrapping: Box())
+            let description = try XCTUnwrap(wrapped.backingViewDescription(bounds: .zero, subtreeExtent: nil))
+            let view = UIView()
+            view.isUserInteractionEnabled = !isUserInteractionEnabled
+            description.apply(to: view)
+            return view
+        }
+
+        XCTAssertTrue(try userInteractionEnabledView(true).isUserInteractionEnabled)
+        XCTAssertFalse(try userInteractionEnabledView(false).isUserInteractionEnabled)
+    }
+
+    func test_convenience() {
```

Comment:
This test probably isn't super useful given it's a one-line passthrough. I'd just remove it

Reply:
Plenty of bugs happen in one line, I think it makes sense to keep this! (e.g., I've accidentally committed a hard coded bool in code like that)

Reply:
Removed.

Reply:
I somehow missed your comment, KB! I'll restore it, but make it a bit less cluttered.